stdlib/public/core/SmallString.swift

```diff
@@ -353,13 +353,22 @@ extension _SmallString {
   //
   @_effects(readonly) // @opaque
   @usableFromInline // testable
-  internal init(taggedCocoa cocoa: AnyObject) {
+  internal init?(taggedCocoa cocoa: AnyObject) {
```

Comment:
ABI change! We probably need to add this initializer as `@_aEIC`, and mark the old one as unavailable.

Reply:
Argh, you’re right, it’s usableFromInline I missed that. Thanks!

Reply:
We discussed this and I'm pretty sure we can get away with it: nothing actually inlines this